java/arcs/core/host/ParticleSpec.kt

```diff
@@ -0,0 +1,9 @@
+package arcs.core.host
+
+/**
+ * A [ParticleSpec] consists of the information neccessary to instantiate a particle
+ * when starting an arc.
+ * [particleName] is human readable name of the Particle in the recipe.
+ * [location] is either a fully qualified Java class name, or a filesystem path.
+ */
+data class ParticleSpec(val particleName: String, val location: String)
```

Comment:
question - in TS equivalent, particle spec has all handle connections as data members, is this a preliminary here that particle doesn't have this info, or is this a way to avoid cyclic dependencies, or else?

Reply:
I did it was a way to avoid cycles. It may be necessary later, but I decided not to "port" the typescript, but start from Shane's architecture doc, and include only the minimum information necessary for it, rather than all of the fields the TS runtime has, many of which are used during recipe resolution, by the inspector, or by the PEC.